src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -5,6 +5,8 @@
 
 // Klasa DefaultCountingOutRhymer może teraz używac obydwu implementacji
 
+// Wszystkie lamia te zasade, brak im wspolnego interfejsu
```

Comment:
Only one rhymer breaks insulation between abstraction and implementation hierarchies (explicit reference to IntLinkedList type)